packages/manager/apps/vrack-services/mock/vrack-services/get-vrack-services.json

```diff
@@ -429,5 +429,51 @@
       "subnets": []
     },
     "updatedAt": "2023-03-10T09:10:00Z"
+  },
+  {
+    "checksum": "e65fa90ceffa95201a49c29b3c031e11",
+    "createdAt": "2024-02-27T17:43:15.225504Z",
+    "currentState": {
+      "displayName": "SUB-test",
+      "productStatus": "DRAFT",
+      "region": "LABEUPRE",
+      "subnets": [
+        {
+          "cidr": "10.0.0.0/24",
+          "displayName": null,
+          "serviceEndpoints": [],
+          "serviceRange": {
+            "cidr": "10.0.0.0/29",
+            "remainingIps": 3,
+            "reservedIps": 5,
+            "usedIps": 0
+          },
+          "vlan": null
+        }
+      ],
+      "vrackId": "pn-9000030"
+    },
+    "currentTasks": [],
+    "id": "vrs-d18-qi7-5lk-is2",
+    "resourceStatus": "READY",
+    "targetSpec": {
+      "displayName": "SUB-test",
+      "subnets": [
+        {
+          "cidr": "10.0.0.0/24",
+          "displayName": null,
```

Comment:
This vrack services is not used I think

Reply:
it is used here => https://github.com/ovh/manager/pull/11293/files#diff-2a48f24cf860380cfed45c7d55c10c1207e6c6904cfe0679d99ff5d26d6d36c4R133

Reply:
ok good